.github/workflows/pull-request.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
       - name: PRs should have at least one qualifying label
         uses: docker://agilepathway/pull-request-label-checker:latest
         with:
-          one_of: chore,fix,bugfix,bug,kind/bug,feature,enhancement,kind/enhancement,dependencies
+          one_of: kind/chore,kind/fix,kind/feature,kind/dependency,kind/documentation,kind/refactor,kind/skip-release-notes,kind/epic,kind/poc
```

Comment:
why is it acceptable to have a PR with kind/epic or kind/poc? IMO those should never be merged. WDYT?

Reply:
yes and no... when a PR closes an epic issue, which label should be used then for the PR?
e.g. "replacement of docker with oras" could be refactoring or epic? Or major API restructuring...

Reply:
IMO a PR that closes an epic doesnt need a kind/epic level, especially because there should never be a single PR that closes an entire epic.